sources/Common/include/Log.h

```diff
@@ -36,8 +36,9 @@ class Log {
    * - one logger with the name of the program, containing logs from dynaflow launcher
    * - one logger named dynawo.log, containing the rest of dynawo logs
```

Comment:
comment not right any more. Also missing comment for parameter options from previous develpments

Reply:
done